### PR TITLE
Selenium tests: Make tests in session_ui not dependent one from each other

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Depends on #472

### CODE REVIEW

**Comment by JohanLorenzo** [comment]
Set ups and tear downs are now explicitly detailed, allowing us to run one test without needing the result from another. I also seized the occasion to enforce the POP in the remaining tests. We should not loose functional coverage between the old tests and the new ones, but another pair of eyes would help me checking that :smiley: .

r? @npark-mozilla

File: test/selenium/lib/make_suite.js [hunk 1]
```diff
@@ -3,33 +3,50 @@
...
     });
-  });
-}
+  },
+
+  browserCleanUp() {
+    return this.app.clear()
+    .then(() => this.app.init(),
+          () => this.app.init());
+  },
```
Comment (nojunpark) [inline test/selenium/lib/make_suite.js:42]: just curious, why the init() called twice here? i'm not so familiar with the arrow notation.

Comment (JohanLorenzo) [inline test/selenium/lib/make_suite.js:42]: `then()` can take 2 parameters: the first one is a callback when things are going fine, and the second one when it failed. Here I wanted to run the init() even if `clear()` failed (for instance, when you run a test in the middle, and there is nothing to clear). You're right on one point, though: things are odd written this way. I'll add a comment explaining why things are written twice in this context.

**Comment by nojunpark** [comment]
other than the need for a comment, everything looks good to me, r+

**Pull request merged**